.github/workflows/workflows.yaml

```diff
@@ -0,0 +1,26 @@
+name: Keycloak FranceConnect CI
+
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Set current version
+        run: mvn -B versions:set -DnewVersion=${GITHUB_REF##*/}
+      - name: Build with Maven
+        run: mvn -B clean verify
+      - name: Upload jar
```

Comment:
Is it a good idea to upload all versions? (snapshot and release) @micedre

Reply:
I don't think we need to upload anything here.